nservicebus/azure-service-bus/performance-tuning_body_asb_[7,).partial.md

```diff
@@ -0,0 +1,61 @@
+## Tuning send performance outside the scope of a handler
+
+Only settings at the ASB SDK level impact this scenario, more specifically
```

Comment:
"The following ASB SDK settings impact out of handler context sending operations"?